profiles/coreos/base/package.accept_keywords

```diff
@@ -80,6 +80,10 @@ dev-util/checkbashisms
 # https://github.com/golang/go/issues?q=milestone%3AGo1.4.3
 =dev-lang/go-1.4.3 ~amd64
 
+# 1.0.2g contains some security fixes.
+# https://bugs.gentoo.org/show_bug.cgi?id=575548
+=dev-libs/openssl-1.0.2g ~amd64 ~arm64
```

Comment:
Actually, the ebuild doesn't have other changes so keywords are correct and this is doing nothing.